Hyper/MarchingCubes/ScalarFieldGenerator.cs

```diff
@@ -34,6 +34,14 @@ public ScalarFieldGenerator(int seed, int octaves = 3, float initialFreq = 0.25f
         _maxAmp = AvgElevation = GetMaxAmp();
     }
 
+    /// <summary>
+    /// Generates a 3d scalar field of voxels.
+    /// </summary>
+    /// <param name="width"></param>
+    /// <param name="height"></param>
+    /// <param name="depth"></param>
+    /// <param name="position"></param>
```

Comment:
maybe describe the `position` parameter, and also indicate along which axes the `width`, ... lengths are measured